src/pykeen/datasets/analysis.py

```diff
@@ -366,7 +340,7 @@ def get_relation_pattern_types_df(
     df = df[(df["support"] >= min_support) & (df["confidence"] >= min_confidence)]
 
     if drop_confidence:
-        df = df[[triple_analysis.RELATION_ID_COLUMN_NAME, "pattern"]].drop_duplicates()
+        df = df[[triple_analysis.RELATION_ID_COLUMN_NAME, "pattern", "support", "confidence"]].drop_duplicates()
```

Comment:
this would now keep the confidence column when you select `drop_confidence` 😅

Reply:
Let's call it a jetlag oversight. I think we should be keeping confidence by default to help make nicer charts

Reply:
Fixed in d5108ace

Reply:
But maybe dropping by default makes sense for visualization. I didn't have an idea for this one yet

Reply:
Getting a smaller output df by default, and only adding "extra information" on request might be useful?